pkg/version/version.go

```diff
@@ -18,7 +18,7 @@ Also https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-versi
 **/
 
 // The main version number that is being run at the moment.
```

Comment:
let's update the version in a separate commit which also updates the CHANGELOG, and is has it's own isse